tests/containers/validate_btrfs.pm

```diff
@@ -30,9 +33,13 @@ sub _btrfs_fi {
 }
 
 sub _sanity_test_btrfs {
-    my ($rt, $dev_path) = @_;
-    my $dockerfile_path = '/root/sle_base_image/docker_build';
-    my $btrfs_head      = '/tmp/subvolumes_saved';
+    my ($rt, $dev_path, $img) = @_;
+    my $dockerfile_path = "~/sle_base_image/docker_build";
+    unless (-d $dockerfile_path) {
+        script_run "mkdir -p $dockerfile_path";
+    }
+    assert_script_run("echo -e 'FROM $img\\nENV WORLD_VAR Arda' > $dockerfile_path/Dockerfile");
```

Comment:
can you explain what this `ENV WORLD_VAR Arda` is for ?

Reply:
it is nothing but just an env variable assigned to the container image. it is something that we also use in the other places just to build the minimum image. Before the Dockerfile was presented in the module, because of some other module had already created. I just copy-paste it.